040-our-benefits.md

```diff
@@ -36,6 +36,46 @@ TEN7 covers 100% of the employee's monthly premium for [2021: Delta Dental PPO p
 
 TEN7 covers 100% of the employee's monthly premium for group vision coverage on the [2021: MN Premier Enhanced Eyewear Plan Opt II B](https://s3.amazonaws.com/hawaiian-ice/production/public/carriers/ca/bcbs/plans/2019/bcbs_mn_premier_enhanced_eyewear_plan_opt_ii_b.pdf) through Blue Cross Blue Shield.
 
+## Life Insurance
+
+You’ll receive [$50,000 Basic Life insurance](https://s3.amazonaws.com/hawaiian-ice/production/public/carriers/ca/principal/plans/2017/principal_mn_principal_life_50000.pdf) from Principal. You may purchase additional coverage at your own expense.
+
+## 401(k)
+
+Upon hire, you can opt in to our 401(k) plan that has a 4% match, and no vesting period. You can also [roll over funds](https://success.guideline.com/hc/en-us/articles/360007550292-Rollover-other-accounts-into-your-Guideline-401-k-) from other eligible accounts. When determining your 401(k) contribution amount, take into account what you’ve contributed during the calendar year so far, so you don’t go over the IRS contribution maximum.
+
+[Guideline](https://www.guideline.com/) can help you with 401(k) issues as well.
+
+## Gusto Cashout
+
+This is a free, no-fee way to access up to $500 between paydays. Use [Cashout](https://app.gusto.com/ten-7-interactive-llc/employee/cashout/overview) to cover an unexpected expense or manage large bills.
+
+## Technology Stipend
+
+As a distributed agency, we know how important it is to have reliable technology. We expect you to have current and working equipment so you’re able to effectively do your job.
+
+We give you an annual $1,500 stipend to use towards technology or work-related services that best suits your needs. The stipend will be deposited on your [PEX](https://www.pexcard.com/) card on a monthly basis, beginning the first full month following your hire date. Please note:
+
+* The equipment or service you buy must be wholly funded by the stipend (i.e. you can’t use a $1,500 stipend towards a $3,000 computer).
+* If you’re a new employee, you may advance ahead on the stipend to purchase a major piece of equipment.
```

Comment:
@danimadelman i would change this line to: If you’re a new employee, you may request an advance to purchase a major piece of equipment.